app/assets/stylesheets/posts.sass

```diff
@@ -0,0 +1,40 @@
+// Place all the styles related to the Posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+/* ADD POST MODAL */
+#addPostPopup 
```

Comment:
ids are a code smell for me, they inherently can only be used once and generally you want to think modularly in classes with you html and css, in case you end up needing to re-use elements and their styles. general best practice suggests using classes instead of ids